config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+                                    }
+                                },
+                                {
+                                    "$comment": "An empty 'container' block will be interpreted as type 'none'",
+                                    "type": "null"
+                                }
+                            ]
+                        },
+                        "defectDojoExport": {
+                            "type": "object",
+                            "properties": {
+                                "parameters": {
+                                    "type": "object",
+                                    "$comment": "This options will directly be sent as parameters to DefectDojo's import/reimport endpoints"
+                                }
+                            }
+                        }
+                    }
+                },
+                {
+                    "$comment": "@FIX: Some templates reference an empty 'general' block. Adding this to ensure proper validation; otherwise, it will default to the 'object' type and fail validation",
+                    "type": "null"
+                }
+            ]
+        },
+        "scanners": {
+            "type": "object",
+            "additionalProperties": {
+                "oneOf": [
+                    {
+                        "type": "object",
+                        "description": "ZAP Scanner",
+                        "properties": {
+                            "apiScan": {
+                                "type": "object",
+                                "properties": {
+                                    "apis": {
+                                        "type": "object",
+                                        "properties": {
+                                            "apiUrl": {
+                                                "type": "string",
+                                                "format": "uri"
+                                            },
+                                            "apiFile": {
+                                                "type": "string"
+                                            }
+                                        },
+                                        "oneOf": [
+                                            {
+                                                "required": [
+                                                    "apiUrl"
+                                                ]
+                                            },
+                                            {
+                                                "required": [
+                                                    "apiFile"
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                },
+                                "required": [
```

Comment:
We don't allow empty `apiScan` block

Reply:
It is allowed. See https://github.com/RedHatProductSecurity/rapidast/blob/development/scanners/zap/zap.py#L468. so 'apis' not required.